test/system/700-play.bats

```diff
@@ -411,6 +411,23 @@ _EOF
     run_podman rmi -f userimage:latest
 }
 
+@test "podman kube play --replace external storage" {
```

Comment:
Also, as a general rule, when a test is super-confusing (like this one, WTF is it supposed to test) I like to add header comments like "Regression test for #12345; an external container, such as from buildah, caused a "container name is already in use" failure.

Reply:
Fixed.